### PR TITLE
Updated Menu

### PR DESCRIPTION
**Pull request by Tiki1tan**
Completley revamped menu to be an instance. This gets rid of all the repetetive ugly code in each scence.gd. Created a global_menu.gd so that one's Sound/Mute options carry over between screens. Created a Mute button. plez no break

### CODE REVIEW

File: screens/login_screen/login_screen.gd [hunk 1]
```diff
@@ -2,7 +2,7 @@ extends Control
 
 export (PackedScene) var next_scene
 
-var check = true # set to false to disable connecting to server
+var check = false # set to false to disable connecting to server
```
Comment (RustyBamboo) [inline screens/login_screen/login_screen.gd]: make sure to leave this true

**Comment by RustyBamboo** [comment]
Also make sure to a git pull --rebase DIM35 master

File: screens/login_screen/login_screen.gd (hunk 1 above)
Comment (Tiki1tan) [reply screens/login_screen/login_screen.gd]: oh yeah

**Comment by Tiki1tan** [comment]
ok rebase done

File: project.godot [hunk 2]
```diff
@@ -18,12 +18,13 @@ config/icon="res://icon.png"
...
 global_player="*res://global_player.gd"
 Base="*res://Base.gd"
+global_menu="*res://global_menu.gd"
 
 [display]
 
 window/dpi/allow_hidpi=true
 window/stretch/mode="2d"
-window/stretch/aspect="keep"
+window/stretch/aspect="expand"
```
Comment (RustyBamboo) [inline project.godot]: Did you push the rebase?

Comment (Tiki1tan) [reply project.godot]: yeah

Comment (RustyBamboo) [reply project.godot]: hmmm, why it be saying this should be change

**Pull request merged**